src/commands/debugCommands.ts

```diff
@@ -0,0 +1,206 @@
+      PromptAnswer.YES,
+      PromptAnswer.NO,
+    );
+    if (answer === PromptAnswer.YES) {
+      try {
+        this.telemetry.sendTelemetryEvent("installDeps");
+        const project = this.dbtProjectContainer.findDBTProject(
+          projectContext.uri,
+        );
+        if (project === undefined) {
+          window.showErrorMessage(
+            `Project ${projectContext.label} was not found`,
+          );
+          return;
+        }
+
+        const depsCommand = this.dbtCommandFactory.createInstallDepsCommand(
+          project.projectRoot,
+          project.dbtProfilesDir,
+        );
+        try {
+          const dbtDepsProcess =
+            await this.dbtProjectContainer.dbtClient?.executeCommand(
+              depsCommand,
+            );
+          const depsOut = await dbtDepsProcess.complete();
+          this.terminal.log(`${dbtDepsProcess.formatText(depsOut.toString())}`);
+        } catch (depsError) {
+          console.log(depsError);
+          window.showErrorMessage(
+            "Error installing dbt dependencies for project " +
+              projectContext.label +
+              ". Please check the output tab for more details.",
+          );
+        }
+      } catch (err) {
+        console.log(err);
+        this.telemetry.sendTelemetryError("installDepsError", err);
+      }
+    }
+  }
+
+  async isVsCodeOutdatated() {
+    this.telemetry.sendTelemetryEvent("vscodeOutdatedStep");
+    const currentVersion = version.toString();
+    const minVersion =
+      this.dbtProjectContainer.context?.extension.packageJSON.engines.vscode
+        .toString()
+        .trim()
+        .replace(/^[^<>~]/, "");
+    if (minVersion === undefined) {
+      return false;
+    }
+    return currentVersion < minVersion;
+  }
+
+  async isExtensionOutdated() {
+    this.telemetry.sendTelemetryEvent("versionOutdatedStep");
+    const currentVersion = this.dbtProjectContainer.extensionVersion.toString();
+    const extLatestJson = await fetch(
+      "https://api.github.com/repos/AltimateAI/vscode-dbt-power-user/releases/latest",
+      {
+        method: "GET",
+        headers: {
+          "Content-Type": "application/json",
+        },
+      },
+    );
+    const extLatest = await extLatestJson.json();
+    const latestVersion = extLatest.tag_name.toString();
+    return currentVersion < latestVersion;
+  }
+
+  private async askforDBTInstallation() {
+    const answer = await window.showErrorMessage(
+      `dbt is not installed in this python environment. Do you want to install dbt?`,
+      PromptAnswer.YES,
+      PromptAnswer.NO,
+    );
+    if (answer === PromptAnswer.YES) {
+      const adapters = this.dbtProjectContainer.getAdapters();
+      if (adapters.length === 0) {
+        window.showErrorMessage(
+          "No adapters detected. could not determine dbt adapter. Please install dbt manually.",
+        );
+        // TODO: telemetry
```

Comment:
Add telemetry

Reply:
This command isnt being called yet. i'll remove it into a patch as you suggested.